### PR TITLE
Add more AOT benchmarks to regression bot.

### PR DESCRIPTION
**Pull request by eerhardt**
This should cover:
- Stage1Aot
- Stage1GrpcAot
- Stage2Aot

### CODE REVIEW

File: build/regressions.config.yml [hunk 1]
```diff
@@ -90,7 +90,7 @@ sources:
   - name: PublishedSizeAot
     table: NativeAotBenchmarks
     rules:
-      - include: Stage1Aot
+      - include: Stage*Aot
```
Comment (sebastienros) [inline build/regressions.config.yml:93]: What does `*` mean? ;)

Comment (eerhardt) [reply build/regressions.config.yml:93]: Same thing it means here:

https://github.com/aspnet/Benchmarks/blob/fc052c63779c4faa17cd5bbc13c79509a6b6ad3d/build/regressions.config.yml#L53

Comment (sebastienros) [reply build/regressions.config.yml:93]: Exactly, so I don't think that's what you want. This field is a regular expression. Other example: `ulib|fasthttp|actix|gin|express|node|netty`

Comment (eerhardt) [reply build/regressions.config.yml:93]: Does this not work, like a normal person would expect?

Comment (eerhardt) [reply build/regressions.config.yml:93]: oh, so it isn't a wildcard. but a regex. I need a dot.

Comment (sebastienros) [reply build/regressions.config.yml]: `Stage.Aot`, `Stage(1|2)Aot`, `Stage1Aot|Stage2Aot`, `Stage\dAot`, `Stage.*Aot` ...

Comment (eerhardt) [reply build/regressions.config.yml]: Better?

**Review by sebastienros — APPROVED** [review]

File: build/regressions.config.yml (hunk 1 above)
Comment (sebastienros) [reply build/regressions.config.yml]: my suggestions won't work with grpc obviously, bad me

**Pull request merged**